### PR TITLE
[131] Admin accessibility check

### LINKED ISSUE #131: [Feature] Make admin only accessible by admins
Check if user is admin before allowing them to access admin panel

### PR DESCRIPTION
**Pull request by natewu**
Only admins can access admin panel

Admin UI improvements

### CODE REVIEW

File: app/src/main/java/com/example/nostack/views/StartUp.java [hunk 1]
```diff
@@ -157,6 +158,11 @@ public void onClick(View v) {
         view.findViewById(R.id.AdministratorSignInButton).setOnClickListener(new View.OnClickListener(){
             @Override
             public void onClick(View v){
+                boolean isAdmin = profile.getRole() != null && profile.getRole().contains("admin");
+                if(!isAdmin){
+                    Toast.makeText(getActivity(), "Unauthorized", Toast.LENGTH_SHORT).show();
```
Comment (olsen-adam) [inline app/src/main/java/com/example/nostack/views/StartUp.java:179]: Should we make this message have slightly more context? Such as "Unauthroized to access this information" or something

File: app/src/main/res/layout/fragment_admin_events_dialog.xml
Comment (olsen-adam) [inline app/src/main/res/layout/fragment_admin_events_dialog.xml:1]: We should really me using wrap_content for this, as constraints on smaller/larger screens can get wonky by having pre-defined text widths

**Review by olsen-adam — APPROVED** [review]
We probably shouldn't be using so many pre-defined element sizes, but apart from that 10/10

File: app/src/main/java/com/example/nostack/views/StartUp.java (hunk 1 above)
Comment (natewu) [reply app/src/main/java/com/example/nostack/views/StartUp.java:179]: i think it works,

**Pull request merged**